src/gen.ml

```diff
@@ -86,6 +86,18 @@ let substitute_vars s vars subst_param =
           parami + 1
       in
       loop acc i2 parami tl
+    | SingleText param :: tl ->
+      let (i1,i2) = param.id.pos in
+      let acc, parami =
+        match param.id.label with
+        | None -> failwith "empty label in IN param"
+        | Some label ->
+          Static (sprintf "@@_sqlgg_%s@@" label) ::
```

Comment:
actually proper way to substitute would be to extend this Static/Dynamic machinery (have one more Dynamic flavor)